src/main/java/com/wolox/training/models/User.java

```diff
@@ -0,0 +1,119 @@
+package com.wolox.training.models;
+
+import com.wolox.training.exception.BookAlreadyOwnedException;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * This class represent the User entity
+ */
+@Entity(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(nullable = false)
+    private @NotNull String username;
```

Comment:
Move the annotation validations to a new line, like: 
```
    @Column(nullable = false)
    @NotNull
    private String username;
```

Reply:
Solved, I idented @NotEmpty